.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,48 @@
+name: Build
+
```

Comment:
is there any workflow or file we are basing this off of? if so, should we add a reference to it?